update-data.sh

```diff
@@ -39,6 +42,24 @@ for VER in ${DRIVER_VERSIONS}; do
             NVIDIA_ARCH=x86_64
         fi
 
+        if [ ${ARCH} == aarch64 ]; then
+            if [ ${MAJOR_VER} -lt 470 ]; then
+                continue
+            elif [[ ${TESLA_VERSIONS} == *${VER}* ]]; then
+                continue
+            elif [[ ${VULKAN_VERSIONS} == *${VER}* ]]; then
+                continue
+            else
+                if [ ${VER} == 510.39.01 ]; then
+                    continue
+                elif [ ${VER} == 495.46 ]; then
+                    continue
+                elif [ ${VER} == 470.94 ]; then
+                    continue
```

Comment:
Why are each of these explicitly excluded? Each link to valid URLs.
510.39.01: https://www.nvidia.com/download/driverResults.aspx/184912/en-us/
495.46: https://www.nvidia.com/Download/driverResults.aspx/184249/en-us/
470.94: https://www.nvidia.com/Download/driverResults.aspx/184164/en-us/

Reply:
They were missing from the URLs the script was generating. I added a new URL that includes those drivers.